Resources/doc/reference/architecture.rst

```diff
@@ -325,7 +325,7 @@ To do this, you first need to call the ``addChild`` method in your PostAdmin ser
     .. code-block:: xml
 
         <!-- app/config/config.xml -->
-        <service id="sonata.news.admin.post" class="Sonata\NewsBundle\Admin\PostAdmin">
+        <service id="sonata.news.admin.post" class="AppBundle\Admin\PostAdmin">
```

Comment:
> Just re-read the doc. Well, you are talking about your define class. Maybe just rename it to other thing than PostAdmin to avoid confusion?

Indeed, very good idea!